static/src/javascripts/projects/commercial/modules/article-body-adverts.js

```diff
@@ -37,21 +41,12 @@ const insertAdAtPara = (para, name, type, classes, sizes) => {
 		});
 };
 
-let previousAllowedCandidate;
-
-// this facilitates a second filtering, now taking into account the candidates' position/size relative to the other candidates
-const filterNearbyCandidates = (maximumAdHeight) => (candidate) => {
-	if (
-		!previousAllowedCandidate ||
-		Math.abs(candidate.top - previousAllowedCandidate.top) -
-			maximumAdHeight >=
-			adSlotClassSelectorSizes.minBelow
-	) {
-		previousAllowedCandidate = candidate;
-		return true;
-	}
-	return false;
-};
+const filterNearbyCandidates = isInVariantSynchronous(
+	spacefinderOkr1FilterNearby,
+	'variant',
+)
+	? fnFixed
```

Comment:
Annoying & non-blocking comment: even though this code will only be around for two weeks it might be worth giving these two modules and functions more descriptive names e.g. `filter-nearby-candidates-broken.ts`, `filterNearbyCandidatesBroken`, etc.

Reply:
An excellent point @zekehuntergreen, I'll update :)